framework_tvb/tvb/adapters/uploaders/brco_importer.py

```diff
@@ -56,13 +56,16 @@ class BRCOImporterModel(UploaderViewModel):
         doc='The Connectivity for which these annotations were made'
     )
 
+    def temporary_paths(self):
```

Comment:
Can't this list be computed with introspection from all attributes of type TraitedUploadField?